README.md

```diff
@@ -1,45 +1,125 @@
 
+```bash
+$ git checkout master
+$ git pull upstream master && git push origin master
 ```
-$ yarn build
+
+Create a branch to work on.
+
+```bash
+$ git checkout -b your-branch
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+Now you can start making changes! See [Building and viewing the site locally](#building-and-viewing-the-site-locally) for visualizing your changes.
 
-### Versioning
+### Submit a pull request
 
+Once you are ready to submit your changes, create a pull request.
+
+```bash
+git push -u origin your-branch
 ```
-$ yarn run docusaurus docs:version 1.1.0
-```
 
-This command creates a copy of /docs and tags with the specified version
+This will create the branch on your GitHub project fork.
+
+**Note** that the -u flag only needs to be used the first time you create a pull request for this branch.
+
+Next, visit the project fork in your GitHub account and locate the new branch, and click "Compare and pull request".
+
+Ensure the "base fork" points to the correct repository and branch and provide a clear title and explanation of the changes with links to any issues that are being fixed by the change.
+
+Then click "Create pull request".
+
+Refer to [Rob Allen's DevNotes](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/) for details on the contribution process.
 
-### Deployment
+## Building and viewing the site locally
 
+The website is built using [Docusaurus 2](https://v2.docusaurus.io/).
+
+Check out their [installation requirements](https://v2.docusaurus.io/docs/installation#requirements) for tooling prerequisites.
+
+### Initialize yarn
+
+The first step is to initialize the use of Yarn to build the site locally.
+
+From your terminal, change directory to the project root and run:
+
+```bash
+$ yarn install
 ```
-$ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
+
+Read the [yarn CLI docs](https://classic.yarnpkg.com/en/docs/cli/) if you want to learn what yarn is doing.
+
+### View the site locally
+
+You can view the site via localhost in your browser by running this command from inside the project:
+
+```bash
+$ yarn start
 ```
 
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+This command starts a local development server and opens up a browser window.
+
+This will fail if there are source code errors or references to relative paths that do not exist.
+
+Most changes to the site are reflected instantly without having to restart the server. Installing a new plugin will require you to restart.
+
+## When will my changes be reflected in docs.temporal.io?
+
+You will have a chance to preview the changes in your pull request. Click "Details" next to the Netlify deploy-preview check.
+
+![Netlify build preview](static/img/readme/netlifypreview.png)
```

Comment:
It seems nit picky but the image has a link in the picture which actually confused me enough that I tried to click it. Can you just take this pic without the link at the end.